src/main.rs

```diff
@@ -105,7 +105,19 @@ fn main() {
                 .long("device")
                 .help("Direct device assignment parameter")
                 .takes_value(true)
-                .min_values(1),
+                .min_values(0),
+        )
+        .arg(
+            Arg::with_name("vunet")
+                .long("vunet")
+                .help(
+                    "Network parameters \"mac=<mac_addr>,\
+                     sock=<socket_path>, num_queue_pairs=<number_of_queue_pairs>,\
+                     queue_size=<size_of_each_queue>, num_vectors=<number_of_vectors>,\
+                     server=<is_server>\"",
+                )
+                .takes_value(true)
+                .min_values(0),
```

Comment:
I'm not sure you can combine .min_values(0) with .takes_value(true)

Reply:
I checked the validator for app args in link https://github.com/clap-rs/clap/blob/6ba47727fce90067a785e84abcb686044fb4fd54/src/app/validator.rs, that the only thing about takes_value and min_values is as this :

if a.takes_value() && !min_vals_zero && ma.vals.is_empty() {
...
}

which means if takes_value specify with true, min_vals is not 0, but the vals actually setup with " " or ' ', it will be invalidate. It does not say if takes_value is true, min_values could not be setup with 0. Thus, I think the setting in the code above is reasonable. :-)

Reply:
Passing "--vunet" without anything after it is not very useful hence this should also be `.min_values(1)`

Reply:
Yes, sensible! Update is done in my local version.